README.md

```diff
@@ -155,7 +155,7 @@ Linux kernel version 3-16.0-rc5)
 
 ```
 $ sudo apt-get install zlib1g-dev libglib2.0-dev libpixman-1-dev libfdt-dev \
-		       libc6:i386 libstdc++6:i386 libz1:i386 cscope
+		       libc6:i386 libstdc++6:i386 libz1:i386 cscope netcat
```

Comment:
@pascal-brand-st-dev I understand why you are commenting here. May I suggest that we just add it to Juno-section for now, since that in fact is the only location where we have stated apt-get dependencies (previously we had it in the setup_xyz_... scripts). I would like that we do a general update of our documentation after I'm back from the GP conf. Like for example, put apt-get pre-req in one section. Remove build instructions from manifest.git. Check and update optee_design.md (some things are outdated).